core/commonKotlin/src/LocalDate.kt

```diff
@@ -121,6 +128,10 @@ public actual class LocalDate actual constructor(public actual val year: Int, pu
         return total - DAYS_0000_TO_1970
     }
 
+    internal actual fun toEpochDaysLong() : Long {
+        return toEpochDays().toLong()
+    }
```

Comment:
```suggestion
    internal actual fun toEpochDaysLong(): Long = toEpochDays().toLong()
```
Style: consistency with the other file, no space before `:`.